tests/bearlib/languages/documentation/DocBaseClassTest.py

```diff
@@ -259,6 +259,102 @@ def test_DocBaseClass_extraction_PYTHON3_4(self):
             list(DocBaseClass.extract(data, 'PYTHON3', 'default')),
             [])
 
+    def test_DocBaseClass_instantiate_padding_PYTHON3_5(self):
+        data = ['\n', '""" documentation in single line """\n',
+                '\n', '\n', 'print(1)']
+
+        docstyle_PYTHON3_default = DocstyleDefinition.load('PYTHON3',
+                                                           'default')
+
+        for doc in DocBaseClass.extract(data, 'PYTHON3', 'default'):
+            self.assertEqual([doc.top_padding, doc.bottom_padding],
+                             [1, 2])
+
+    def test_DocBaseClass_instantiate_padding_PYTHON3_6(self):
+        data = ['\n', '\n', '""" documentation in single line """\n',
+                '\n', '\n', 'print(1)\n']
+
+        docstyle_PYTHON3_default = DocstyleDefinition.load('PYTHON3',
+                                                           'default')
+
+        for doc in DocBaseClass.extract(data, 'PYTHON3', 'default'):
+            self.assertEqual([doc.top_padding, doc.bottom_padding],
+                             [2, 2])
+
+    def test_DocBaseClass_instantiate_padding_inline_PYTHON3_7(self):
+        # To test that bottom_padding sets to nothing if docstring is
+        # followed by inline docstring.
```

Comment:
we have such a case in DocStyleBear.
I don't wanna break the existing tests :wink: 
https://github.com/coala/coala-bears/blob/master/tests/documentation/test_files/DocumentationStyleBear/bad_file2.py.test#L21